### PR TITLE
docs: removed unnecessary files and add documentation about Iterable

### PR DESCRIPTION
**Pull request by Arfey**

### CODE REVIEW

File: docs/web_use.rst [hunk 1]
```diff
@@ -1,2 +1,256 @@
...
+    import trafaret as t
+    from trafaret.keys import subdict
+
+
+    def author_validation(data):
+        """Test that first author exist in list of all authors."""
+        
+        if data['first_author'] not in data['authors']:
+            raise t.DataError('first_author must exist in authors')
+
```
Comment (Deepwalker) [inline docs/web_use.rst]: you should return something that will be stored in 'authors' key.

Comment (Arfey) [reply docs/web_use.rst]: But i don't want. I want only validate relation between two fields.

Comment (Deepwalker) [reply docs/web_use.rst]: Then you are using wrong instrument. subdict will remove both keys from result set and will return `'authors': None` instead.
To make just this check it is better to check it after, like `t.Dict(jsdkahfjkashfksajfh) & check_that_author_is_in_authors`.

Comment (Arfey) [reply docs/web_use.rst]: Okey, thx i'll try to rewrite it.

**Pull request merged**